modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceStateManager.java

```diff
@@ -57,8 +56,7 @@ public class ServiceStateManager {
     private DatabaseServiceStateManagementTaskStorage taskStorage;
     @Resource
     private DatabaseDeployServiceStorage deployServiceStorage;
-    @Qualifier("xpanseAsyncTaskExecutor")
-    @Resource
+    @Resource(name = "xpanseAsyncTaskExecutor")
```

Comment:
can we use the name from here - https://github.com/swaroopar/osc/blob/191b44b81a1113b5675a64260e2b046aac918c36/modules/async/src/main/java/org/eclipse/xpanse/modules/async/TaskConfiguration.java#L21